lang/syn/src/idl/file.rs

```diff
@@ -513,6 +513,7 @@ fn idl_accounts(
         .iter()
         .map(|acc: &AccountField| match acc {
             AccountField::CompositeField(comp_f) => {
+                println!(comp_f.symbol);
                 let accs_strct = global_accs
                     .get(&comp_f.symbol)
                     .expect("Could not resolve Accounts symbol");
```

Comment:
With this implementation we would always log the symbol which is not what armani meant. We should only log it inside the `.expect` call.

Reply:
@paul-schaaf Oh got it, thanks for the feedback!